projects/common/src/utilities/validators/regex-validator.test.ts

```diff
@@ -0,0 +1,14 @@
+import { isValidRegex } from './regex-validator';
+
+const VALID_REGEX = ['2+2', '(2+2) * 3*9'];
+const INVALID_REGEX = ['\\', '['];
```

Comment:
why are these outside the test description? please move them inside.

Reply:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/60038750/174863086-5ae0e7bb-f17c-4691-b13e-a33fcca31c45.png">
Followed this convention so that's why. Updating